build_in_methods_iterators/homework.py

```diff
@@ -14,7 +17,10 @@ def task_1_fix_names_start_letter(data: DT) -> DT:
         fix_names_start_letters([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}])
         >>> [{'name': 'Alex', 'age': 26}, {'name': 'Denys', 'age': 89}]
     """
-    pass
+    for DT in data:
```

Comment:
change please `DT` to correct name of local variable. For example `item`

Reply:
like this?